openapi/data/fields.py

```diff
@@ -222,9 +222,21 @@ def dump(self, value):
 
 
 class DecimalValidator(NumberValidator):
-    def __call__(self, field, value, precision):
+    def __call__(self, field, value):
         try:
             value = Decimal(value)
         except TypeError:
             raise ValidationError(field.name, '%s not valid Decimal' % value)
-        return super().__call__(field, value, precision)
+        return super().__call__(field, value)
+
+
+class BoolValidator(Validator):
+
+    def __call__(self, field, value):
+        value = str(value).lower()
+        if value not in ('true', 'false'):
+            raise ValidationError(field.name, '%s not valid' % value)
+        return value == 'true'
+
+    def dump(self, value):
+        return str(value).lower()
```

Comment:
I don't think this is right.
return ``bool(value)`` makes more sense to me

Reply:
the dump should result in a json valid format, no? In this case, `"true"` or `"false"`

Reply:
a boolean is a valid JSON value.
It needs to be `True` or `False`

Reply:
I dont think that `True` or `False` are valid json values.

``` 
A JSON value MUST be an object, array, number, or string, or one of
   the following three literal names:

      false null true
```

https://tools.ietf.org/html/rfc7159

Reply:
@jklepatch they don't need to be Json type, they need to be types python know how to convert to JSON.
@daniloaf, please return a boolean. It is the correct return type.
These values are part of the dictionary which is subsequently serialised via JSON.
```python
json.dumps({'whatever': True})
'{"whatever": true}'
```

Reply:
tests should be able to help here too. Try seruializing after `dump`

Reply:
So I believe some validators are wrong, like DateTimeValidator as it returns the datetime isoformat. Should I change those?